benchmark_runner/common/template_operations/template_operations.py

```diff
@@ -124,7 +124,7 @@ def generate_yamls_internal(self, scale: str = None, scale_node: str = None, red
             'workload_extra_name': self.__workload_extra_name,
             'standard_output_file': self.__standard_output_file,
             'standard_template_file': self.__standard_template_file,
-            'scale': int(scale)+1,
+            'scale': scale,
```

Comment:
Why isn't this
```
str(int(scale)+1)
```
?

Reply:
We cannot use int just string, revert it back to scale